caddyhttp/httpserver/replacer.go

```diff
@@ -309,6 +309,12 @@ func (r *replacer) getSubstitution(key string) string {
 	case "{dir}":
 		dir, _ := path.Split(r.request.URL.Path)
 		return dir
+	case "{user}":
+		user, _ := r.request.Context().Value(RemoteUserCtxKey).(string)
+		if user == "" {
+			user = "-"
```

Comment:
@mholt  doh!  well I appreciate your passion as well 😄,  and I was all fired up! , but I see what you mean.   Yeah lines 314-315 can go.